src/components/ProductItem/index.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { currencyFormat } from "../../utils/currency";
+
+import "./styles.css";
+
+const ProductItem = ({ product }) => {
+  const { name, images, description, price, categories } = product;
+  return (
+    <div className="card product-box">
+      <div
+        style={{
+          minHeight: "230px",
+          maxHeight: "230px",
+        }}
+      >
+        <img
+          src={images[0]}
+          className="card-img-top"
+          alt={name}
+          style={{ width: "100%", height: "15vw", objectFit: "contain" }}
+        />
+      </div>
+      <div className="card-body">
+        <h5 className="card-title fw-bold" style={{ minHeight: "80px" }}>
+          {name.substring(0, 80) + "..."}
```

Comment:
I see what you're doing here, maybe a better approach could be managing it with CSS, try looking up how to truncate text or use line-clamp to do it

Reply:
I fixed it